### DISCUSSION TITLE
如何显示 user agent ？

### DISCUSSION THREAD

**Discussion by iDvel** [walinejs/waline#753] [Q&A] [answered]
看到了一个 `DISABLE_USERAGENT` 参数，默认就是 `false`，但是并没有显示用户代理呀。

**Comment by lizheming** [comment, marked as answer]
提供一下博客地址看一下

**Reply by iDvel** [reply]
https://dvel.me
请问默认就是不显示吗，还是需要设置什么。

**Reply by iDvel** [reply]
之前没有设置这个参数，后来设置了 `false` 也不行，现在删除了也不行。
不知道是应该在哪里调整 UA 的显示呢？我看别人都可以显示。。。

**Reply by lizheming** [reply]
可能是有点问题，晚点我看下吧

**Reply by lizheming** [reply]
中间加需求加出 Bug 了，已修复，更新到 @waline/vercel@1.5.1 就可以啦，感谢反馈~~~
